### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Mwapsam**
### Set up a "Hello World" project with two apps

**In this activity I created an API and the frontend of the hello-rails app. To do this I did the following:**

- _Postgres database set up._

- _Created a table for storing the messages and created 5 different greetings._

- _Created an API endpoint that selects a random greeting from your table_ 

- _Created the App component with react-router._

- _Created the Greeting component that displays a greeting. Set it up as a route in the App component_.

- _Created a store, an action, and a reducer that will connect to the API endpoint to get the random greeting._

- _Displayed the random greeting in the Greeting component._

[Frontend](https://github.com/Mwapsam/Hello-rails-frontend/pull/1)

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @Mwapsam,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

#### 1. You do not have a descriptive README
#### 2. Your README does not contain a link to your associated Frontend app

#### NOTE: Ensure there is a Pull Request in the frontend repo

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+
+Message.create(text: "Hello, world! 1")
+Message.create(text: "Hello, world! 2")
+Message.create(text: "Hello, world! 3")
+Message.create(text: "Hello, world! 4")
+Message.create(text: "Hello, world! 5")
```
Comment (ahmetbozaci) [inline db/seeds.rb]: ### OPTIONAL
- [ ] You can create a different greetings instead write numbers end of the text.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @Mwapsam  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Kindly update your readme file.
- License link is broken.
- You added a fronted pr link. Kindly change it to the repository link.
- You don't have tests in this project so you can delete this section.
- Also you work on the develop branch when creating the readme file. Next time please work on the feature branch.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Mwapsam, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Descriptive Readme file

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**